src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Formatting/CSharpFormattingPassBase.cs

```diff
@@ -367,18 +377,31 @@ bool IsImplicitStatementStart()
             return false;
         }
 
-        bool IsInHtmlAttributeName()
+        bool IsInBoundComponentAttributeName()
```

Comment:
This method name was doing a poor job of describing what it did. Also expanded the comment below to be clearer.